gulpfile.js

```diff
@@ -64,13 +64,28 @@ gulp.task('lint-test', function() {
     .pipe($.jshint.reporter('fail'));
 });
 
+function getBanner() {
+  var banner = ['/**',
+    ' * <%= pkg.name %> - <%= pkg.description %>',
+    ' * @version v<%= pkg.version %>',
+    ' * @link <%= pkg.homepage %>',
+    ' * @license <%= pkg.license %>',
+    ' */',
+    ''].join('\n');
+
+  return _.template(banner)({pkg: manifest});
```

Comment:
It didn't work without adding lodash.

true.. this was a result of a lame copy-pasta from that gulp-header library I was playing with :stuck_out_tongue_winking_eye: